x-pack/plugins/cloud_security_posture/public/pages/findings/findings.tsx

```diff
@@ -107,14 +147,7 @@ export const Findings = () => {
         <Route
           exact
           path={cloudPosturePages.findings.path}
-          render={() => (
-            <Redirect
-              to={{
-                pathname: findingsNavigation.findings_default.path,
-                search: location.search,
```

Comment:
before the change location search was also passed, but it isn't happening in `FindingsTabRedirecter`. Do we need to preserve this or it wasn't needed anyway?

Reply:
good catch!